server/pulp/plugins/loader/api.py

```diff
@@ -81,7 +82,20 @@ def list_content_types():
     :rtype: list of str
     """
     assert _is_initialized()
-    return database.all_type_ids()
+    types_list = _MANAGER.unit_models.keys()
+    legacy_types = database.all_type_ids()
+    types_list.extend(legacy_types)
+    return types_list
+
+
+def list_unit_models():
+    """
+    List the supported unit_models.
```

Comment:
Consider adding a newline under this to separate the description from the params in the docstring.